src/lib/github/addIssuesToRepo.ts

```diff
@@ -16,31 +16,29 @@ export default async function addIssuesToRepo (context: Context, newIssues: Issu
   newIssues.forEach(async ({title, body, authors}) => {
     const assignees = uniq(authors);
 
-    /**
-     * Am up to writing a function that removes checked out or no longer existing issue comments.
-     */
-
-    const currentGHIssueComments = find(ghIssueComment => ghIssueComment.comments === body, ghIssues);
-    const newGHIssueComments = currentGHIssueComments.comments.includes(" - [x] ") || !(currentGHIssueComments.comments === body) ? replaceCommentArray(currentGHIssueComments.comments, body) : currentGHIssueComments.comments;
+    const currentGHIssue = find(ghIssue => ghIssue.title === title, ghIssues);
+    const newGHIssueComments = currentGHIssue.comments.includes("- [x]") || !(currentGHIssue.comments === body)
+     ? newIssueBody(currentGHIssue.comments, body)
+     : currentGHIssue.comments;
 
     const fields = {
       owner,
       repo,
-      body,
+      newGHIssueComments,
```

Comment:
this should be `body:  newGHIssueComments`

Reply:
true